### ISSUE TITLE
MoBi: Remove multiple results per simulation

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSMOSES#9]
Unify behavior with PK-Sim such that only the latest results are stored.

**Comment by msevestre** [comment]
@Yuri05 @PavelBal Should not we move all thos MoBi related issues to the MoBi repo? Maybe add them to the project OSMOSES (that I could create) instead?

**Comment by PavelBal** [comment]
> Should not we move all thos MoBi related issues to the MoBi repo? Maybe add them to the project OSMOSES (that I could create) instead?

I think it's for you and @Yuri05 to decide...

**Comment by StephanSchaller** [comment]
Yes, we could also do it this way… this may also make it easier to categorize the tasks (Kanban style)

**Comment by Yuri05** [comment]
>Should not we move all thos MoBi related issues to the MoBi repo? Maybe add them to the project OSMOSES (that I could create) instead?

Fine with that
